openhcl/diag_proto/src/diag.proto

```diff
@@ -7,6 +7,16 @@ package diag;
 
 import "google/protobuf/empty.proto";
 
+// OpenHCL diagnostics services.
+//
+// Add new methods here, since the diagnostics service in use before this did
+// not handle unknown methods correctly.
+service OpenhclDiag {
+    // Ping the server, validating it is ready for use.
+    rpc Ping(google.protobuf.Empty) returns (google.protobuf.Empty);
+}
+
+// Older methods.
```

Comment:
what's the off-ramp here? are we stuck waiting for the current in-market releases to fall off?

Reply:
Probably if we want to get rid of the old service name, we need to add all the existing methods to the new service and update clients to try both. I think we could do this somewhat transparently via some "rename" functionality in the client and server packages. But it's a fairly minor wart to have this thing sticking around, so I'm not too worried about it right now.

I'll open an issue.

Reply:
#403